donate.html

```diff
@@ -24,7 +24,7 @@
     {% else %}
     <form id="act" name="act"
       class="{% if page.payfment_processor_information.use_tr %}braintree_form{% endif %} {{ templateset.custom_fields.donation_steps }} "
-      method="POST" action="https://350.actionkit.com/act/" accept-charset="utf-8">

```

Comment:
FYI this change was inspired by guidance from AK support on Mar 8. The inspiration was from troubleshooting the analytics "user journey" getting interrupted by CORS policy. Snippet from AK support email:

> I think I see the issue. The "donate.html" template for this page's template set has the "act" form with its action hard-coded to "https://350.actionkit.com/act/":
> 
> https://act.350.org/admin/cms/template/18485/edit/
> 
> In 2020, Chrome rolled out a new default referrer policy, "strict-origin-when-cross-origin":
> 
> https://developers.google.com/web/updates/2020/07/referrer-policy-new-chrome-default
> 
> So even though the donate page is loaded from act.350.org, it posts to 350.actionkit.com, and Chrome only sends "act.350.org" along.
> 
> A simple fix would be to change the form action in the template to "/act/".

Reply:
This makes sense! Thanks @alandmih